CHANGELOG.adoc

```diff
@@ -29,6 +29,7 @@ A release with an intentional breaking changes is marked with:
 ** {issue}642[#642]: Add `driver-type` to retrieve driver type keyword. ({person}dgr[@dgr])
 ** {issue}644[#644]: Deprecate `when-predicate` and `when-not-predicate` macros. See docstrings for guidance on alternatives. These will may be removed from the API at the next release that contains breaking changes. ({person}dgr[@dgr])
 ** {issue}647[#647]: Fix logic bug in `intersects?`. Added tests to test suite. ({person}dgr[@dgr])
+** {issue}649[#649]: When supplied a vector argument for `q-text`, `fill-multi` and `fill-human-multi` now fill fields in the order that fields appear in the vector. Previously, the order was not guaranteed. ({person}dgr[@dgr])
```

Comment:
Dave, I just noticed now you are categorizing your non-doc changes under Docs.

Reply:
OK, just pushed changes. Sorry about that. Didn't notice that there was a separate section for docs.